setup.py

```diff
@@ -484,7 +484,7 @@ def run(self):
         http_user = "wwwrun"
         httpd_service = "apache2.service"
         defaultpath = "/etc/sysconfig/"
-    elif os.path.exists("/etc/debian_version"):
+    elif distro.id() in ("debian", "ubuntu"):
         if os.path.exists("/etc/apache2/conf-available"):
```

Comment:
We can remove the `os.path.exists` check and just always assume under debian-ish systems that `webconfg = "/etc/apache2/conf-available"`. There is no supported version of Debian/Ubuntu where that doesn't exist.

Reply:
Removed it.